src/index.js

```diff
@@ -317,17 +309,14 @@ export default class Headroom extends Component {
       ? `${userClassName} headroom-wrapper`
       : 'headroom-wrapper'
 
+    const { component } = this.props
+
     return (
-      <div style={wrapperStyles} className={wrapperClassName}>
-        <div
-          ref={this.setRef}
-          {...rest}
-          style={innerStyle}
-          className={className}
-        >
+      <component style={wrapperStyles} className={wrapperClassName}>
```

Comment:
This actually doesn't work as intended. It will render `<component>` tag to the DOM. [User-Defined Components Must Be Capitalized](https://reactjs.org/docs/jsx-in-depth.html#user-defined-components-must-be-capitalized)

Reply:
Yes, it should be PascalCase

Reply:
also i think that `tag` would be a better name for this prop. it controls what **tag** will get rendered :)